compiler/next/test/parsing/testParsingQueries.cpp

```diff
@@ -79,16 +79,16 @@ static void test2() {
   std::string modTwoContents = "/* this is a another test */"
                                "a;\n";
   setFileText(ctx, modTwoPath, modTwoContents);
- 
+
   const Module* moduleOne = nullptr;
   const Module* moduleTwo = nullptr;
 
   moduleOne = parseOneModule(ctx, modOnePath);
   moduleTwo = parseOneModule(ctx, modTwoPath);
   assert(moduleOne->numStmts() == 2);
   assert(moduleTwo->numStmts() == 2);
-  ASTNode::dump(moduleOne);
-  ASTNode::dump(moduleTwo);
+  moduleOne->stringify(std::cout, chpl::StringifyKind::DEBUG_DETAIL);
+  moduleTwo->stringify(std::cout, chpl::StringifyKind::DEBUG_DETAIL);
```

Comment:
You can just delete the dump calls in this test, they are leftovers from debugging and IMO aren't really needed once we have tracing working

Reply:
Did you mean to remove it from only `test2`, or all the tests in this file?